packages/jest-message-util/package.json

```diff
@@ -8,10 +8,10 @@
   "license": "MIT",
   "main": "build/index.js",
   "dependencies": {
-    "@babel/code-frame": "^7.0.0-beta.35",
+    "@babel/code-frame": "7.0.0-beta.44",
```

Comment:
last version with node 4 support. Upgrading it to 7 stable breaks the browser build... not sure why TBH, didn't dig into it